tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -262,14 +263,25 @@ func tearDownInstance() error {
 	return nil
 }
 
+// testImageUsed tests that the proper Windows AMI was used
+func testImageUsed(t *testing.T) {
+	describedImages, err := awsProvider.EC2.DescribeImages(&ec2.DescribeImagesInput{
+		ImageIds: []*string{createdInstance.ImageId},
+	})
+	require.NoErrorf(t, err, "Could not describe images with imageID: %s", createdInstance.ImageId)
+	require.Lenf(t, describedImages.Images, 1, "Found unexpected amount of AMIs with imageID %s",
+		createdInstance.ImageId)
+
+	foundImage := describedImages.Images[0]
+	require.Contains(t, foundImage.Name, "Windows_Server-2019-English-Full-ContainersLatest")
```

Comment:
Will this be the same always?

Reply:
It should always contain that substring, plus a suffix including the date